### PR TITLE
Fixed get_by_item_id query with ql parameter

### PR DESCRIPTION
**Pull request by deathlef**

### CODE REVIEW

File: modules/standard/items/items_controller.py [hunk 1]
```diff
@@ -136,13 +136,13 @@ def sort_items(self, search, items):
 
     def get_by_item_id(self, item_id, ql=None):
         if ql:
-            return self.db.query_single("SELECT * FROM aodb WHERE (highid = ? OR lowid = ?) AND (lowql <= ? AND highql >= ?) ORDER BY highid DESC LIMIT 1", [item_id, item_id, ql, ql])
+            return self.db.query_single("SELECT * FROM aodb WHERE (highid = ? OR lowid = ?) AND (lowql <= ? AND highql >= ?) ORDER BY highid = ? LIMIT 1", [item_id, item_id, ql, ql, item_id])
         else:
-            return self.db.query_single("SELECT * FROM aodb WHERE highid = ? OR lowid = ? ORDER BY highid DESC LIMIT 1", [item_id, item_id])
+            return self.db.query_single("SELECT * FROM aodb WHERE highid = ? OR lowid = ? ORDER BY highid = ? LIMIT 1", [item_id, item_id])
```
Comment (Minidodo1) [inline modules/standard/items/items_controller.py:141]: you forgot the third "item_id" param, i.e. "[item_id, item_id, item_id]"
For my Tests i added the third one as item_id.

File: modules/standard/items/items_controller.py [hunk 2]
```diff
@@ -136,13 +136,13 @@ def sort_items(self, search, items):
...
 
     def find_by_name(self, name, ql=None):
         if ql:
-            return self.db.query_single("SELECT * FROM aodb WHERE name = ? AND lowql <= ? AND highql >= ? ORDER BY highid DESC LIMIT 1", [name, ql, ql])
+            return self.db.query_single("SELECT * FROM aodb WHERE name = ? AND lowql <= ? AND highql >= ? ORDER BY highid DESC LIMIT 1", [name, ql, ql, item_id])
```
Comment (Minidodo1) [inline modules/standard/items/items_controller.py:145]: Here you added the 4th param, but didnt change the order by to "ORDER BY highid = ?"
in my tests, i removed the 4th param.

**Review by Minidodo1 — CHANGES_REQUESTED** [review]

**Comment by bigwheels16** [comment]
I think this looks pretty good to me now

**Review by Minidodo1 — APPROVED** [review]
Yup, should work now - tested the same changes in my 3 bots so far, no issues till now 👍

**Pull request merged**